com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.cs

```diff
@@ -1181,6 +1183,13 @@ protected override void Render(ScriptableRenderContext renderContext, Camera[] c
                 HDCamera.CleanUnused();
             }
 
+            // Update the water surfaces
+            var commandBuffer = CommandBufferPool.Get("");
+            UpdateWaterSurfaces(commandBuffer);
+            renderContext.ExecuteCommandBuffer(commandBuffer);
```

Comment:
Pure sanity check, @JulienIgnace-Unity any thought about calling update of water surface with command buffer like this?